src/main/java/de/hpi/bpt/argos/storage/dataModel/event/type/StatusUpdatedEventType.java

```diff
@@ -0,0 +1,231 @@
+import de.hpi.bpt.argos.util.LoggerUtilImpl;
+import de.hpi.bpt.argos.util.PairImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * {@inheritDoc}
+ * This is a special implementation for indicating that an entity has changed its status.
+ */
+public final class StatusUpdatedEventType extends EventTypeImpl {
+	private static final Logger logger = LoggerFactory.getLogger(StatusUpdatedEventType.class);
+
+	public static final String NAME = "StatusUpdatedEventType";
+
+	private static final String TIMESTAMP_ATTRIBUTE_NAME = "Timestamp";
+	private static final String OLD_STATUS_ATTRIBUTE_NAME = "OldStatus";
+	private static final String NEW_STATUS_ATTRIBUTE_NAME = "NewStatus";
+	private static final String CAUSE_EVENT_ID_ATTRIBUTE_NAME = "CauseEventId";
+	private static final String CAUSE_EVENT_TYPE_ID_ATTRIBUTE_NAME = "CauseEventTypeId";
+	private static final String ENTITY_ID_ATTRIBUTE_NAME = "EntityId";
+
+	/**
+	 * This constructor hides the default public one to implement the singleton pattern.
+	 */
+	private StatusUpdatedEventType() {
+
+	}
+
+	/**
+	 * This method sets up the statusUpdatedEventType.
+	 * @param argos - the argos instance
+	 * @return - the statusUpdatedEventType
+	 */
+	public static EventType setup(Argos argos) {
+		EventType instance = getInstance();
+
+		if (instance != null) {
+			return instance;
+		}
+
+		instance = create();
+		registerCustomMapping(argos);
+		return instance;
+	}
+
+	/**
+	 * This method returns the statusUpdatedEventType.
+	 * @return - the statusUpdatedEventType
+	 */
+	public static EventType getInstance() {
+		return PersistenceAdapterImpl.getInstance().getEventType(NAME);
+	}
+
+	/**
+	 * This method sends a statusUpdatedEvent to the eventProcessingPlatform.
+	 * @param entity - the entity, which status was updated
+	 * @param oldStatus - the old status of the given entity
+	 * @param newStatus - the new status of the given entity
+	 * @param statusUpdateTrigger - the event, which caused the status update
+	 * @return - the feedback of the eventProcessingPlatform
+	 */
+	public static EventPlatformFeedback postEvent(Entity entity, String oldStatus, String newStatus, Event statusUpdateTrigger) {
+
+		EventPlatformFeedback feedback = EventFactoryImpl.getInstance().postEvent(getInstance(), TIMESTAMP_ATTRIBUTE_NAME,
+				new PairImpl<>(OLD_STATUS_ATTRIBUTE_NAME, oldStatus),
+				new PairImpl<>(NEW_STATUS_ATTRIBUTE_NAME, newStatus),
+				new PairImpl<>(CAUSE_EVENT_ID_ATTRIBUTE_NAME, statusUpdateTrigger.getId()),
+				new PairImpl<>(CAUSE_EVENT_TYPE_ID_ATTRIBUTE_NAME, statusUpdateTrigger.getTypeId()),
+				new PairImpl<>(ENTITY_ID_ATTRIBUTE_NAME, entity.getId()));
+
+		logger.info(String.format("sending status update event: entity: '%1$s' status: '%2$s' -> '%3$s' : response '%4$s'",
+				entity.getName(),
+				oldStatus,
+				newStatus,
+				feedback.getResponseText()));
+
+		return feedback;
+	}
+
+	/**
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AJavadoc_Style_BP2016)